btr-web/common-layer/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "nuxt-app",
```

Comment:
pnpm can filter on names, so I like to name this after the layer being used, maybe `common-layer`